odoo/tools/image.py

```diff
@@ -75,7 +75,7 @@ def image_resize_image(base64_source, size=(1024, 1024), encoding='base64', file
 
     if image.size != size:
         image = image_resize_and_sharpen(image, size)
-    if image.mode not in ["1", "L", "P", "RGB", "RGBA"]:
+    if image.mode not in ["1", "L", "P", "RGB"]:
```

Comment:
@xmo-odoo Good point. I'm not sure we should continue silently discarding the alpha channel when a developer requests a nonsensical conversion (i.e. what PIL stopped doing), but let's keep it for _stable_ at least.
Apparently PIL still allows it for other formats that don't have true alpha channels, so we only need to cover JPEG so far.